test/extended/openstack/servergroup.go

```diff
@@ -151,10 +155,14 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack pla
 
 				instanceWorkerGroupName := instanceWorkerGroupNameField.(string)
 				o.Expect(instanceWorkerGroupName).NotTo(o.BeEmpty(), "the Server group name should not be the empty string")
-				if workerGroupName == "" {
-					workerGroupName = instanceWorkerGroupName
-				} else {
-					o.Expect(instanceWorkerGroupName).To(o.Equal(workerGroupName), "two Worker Machines have different workerGroupName set")
+
+				// If there are multiple AZs, our workers will belong to separate server groups (one per AZ)
+				if availabilityZoneCount == 1 {
```

Comment:
The assumption here is that there are multiple AZs used by the cluster in case there are more than one which may be wrong (e.g. one AZ is used by the cluster , others are not)

Reply:
Done :+1: